src/current/_includes/releases/v24.3/v24.3.2.md

```diff
@@ -0,0 +1,18 @@
+## v24.3.2
+
+Release Date: December 26, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-2-sql-language-changes">SQL language changes</h3>
+
+- Added the `legacy_varchar_typing` session setting. When set to `on`, type-checking and overload resolution ignore the newly added `unpreferred` overload preference which causes a change in `VARCHAR` typing behavior (see [#133037][#133037]). The default value for `legacy_varchar_typing` is `on`. [#137943][#137943]
+
+<h3 id="v24-3-2-bug-fixes">Bug fixes</h3>
```

Comment:
This bug fix should be removed:

- this maps to https://github.com/cockroachdb/cockroach/pull/133037, which is the [buggy] bug fix that was introduced by v24.3.1, but is being reverted by the "hot fix" change (https://github.com/cockroachdb/cockroach/pull/137919 for 24.3)